play.py

```diff
@@ -148,42 +148,43 @@ def play_aidungeon_2():
 
         while True:
             sys.stdin.flush()
-            action = input("> ")
-            if action.lower() == "restart":
+            action = input("> ").lower()
+            normalized_action = action.lower()
```

Comment:
Any reason you're using `.lower()` twice?

Reply:
Also, `action` shouldn't be lowercased because the action might not be a command, which would cause the input into the story to lose information like proper nouns

Reply:
Derp I totally didn't double check. Transferred the change from my collab instance at midnight so my brain definitely did not function properly 😂

Reply:
Also #137 solved this problem well so I will just remove my change for play.py 😆